llvm/test/CodeGen/AMDGPU/regpressure_printer.mir

```diff
@@ -459,4 +433,55 @@ body:             |
     S_NOP 0, implicit %16
     S_ENDPGM 0
 ...
-
+---
+name:  test_early_clobber_trivial
+tracksRegLiveness: true
+body:             |
+  bb.0:
+    ; RP-LABEL: name: test_early_clobber_trivial
+    ; RP: Live-in:
+    ; RP-NEXT: SGPR  VGPR
+    ; RP-NEXT: 0     0
+    ; RP-NEXT: 0     1      %0:vgpr_32 = V_MOV_B32_e32 42, implicit $exec
+    ; RP-NEXT: 0     1
+    ; RP-NEXT: 0     2      early-clobber %1:vgpr_32 = V_MOV_B32_e32 %0:vgpr_32, implicit $exec
+    ; RP-NEXT: 0     1
+    ; RP-NEXT: 0     1      S_NOP 0, implicit %1:vgpr_32
+    ; RP-NEXT: 0     0
+    ; RP-NEXT: Live-out:
+    %0:vgpr_32 = V_MOV_B32_e32 42, implicit $exec
+    early-clobber %1:vgpr_32 = V_MOV_B32_e32 %0, implicit $exec
+    S_NOP 0, implicit %1
+...
+---
+name:  test_not_early_clobber_trivial
+tracksRegLiveness: true
+body:             |
+  bb.0:
+    ; RPU-LABEL: name: test_not_early_clobber_trivial
+    ; RPU: Live-in:
+    ; RPU-NEXT: SGPR  VGPR
+    ; RPU-NEXT: 0     0
+    ; RPU-NEXT: 0     1      %0:vgpr_32 = V_MOV_B32_e32 42, implicit $exec
+    ; RPU-NEXT: 0     1
+    ; RPU-NEXT: 0     1      %1:vgpr_32 = V_MOV_B32_e32 %0:vgpr_32, implicit $exec
+    ; RPU-NEXT: 0     1
+    ; RPU-NEXT: 0     1      S_NOP 0, implicit %1:vgpr_32
+    ; RPU-NEXT: 0     0
+    ; RPU-NEXT: Live-out:
+    ;
+    ; RPD-LABEL: name: test_not_early_clobber_trivial
+    ; RPD: Live-in:
+    ; RPD-NEXT: SGPR  VGPR
+    ; RPD-NEXT: 0     0
+    ; RPD-NEXT: 0     1      %0:vgpr_32 = V_MOV_B32_e32 42, implicit $exec
+    ; RPD-NEXT: 0     1
+    ; RPD-NEXT: 0     2      %1:vgpr_32 = V_MOV_B32_e32 %0:vgpr_32, implicit $exec
```

Comment:
So the downward tracker also needs to be fixed?

Reply:
Yes, I'm going to fix it in a separate PR, we should have a perfect match of RP in both ways.